README.md

```diff
@@ -61,25 +61,12 @@ Unit tests can be ran locally by running the command in root of the project:
 
 ### E2E tests
 
-E2E tests are ran as part of CI, but can also be ran locally. To run the E2E tests locally, you'll need to do the following:
-- Install Gingko following the intructions [here](https://onsi.github.io/ginkgo/#installing-ginkgo)
-- Clone `subnet-evm` from [Github](https://github.com/ava-labs/subnet-evm) and checkout the correct version as specified in `go.mod` or `scripts/versions.sh`
-
-Next, set up the `avalanchego` build path and `subnet-evm` binary, making sure to install everything in a writeable location (here we use `~/tmp`):
-
-```bash
-cd subnet-evm
-BASEDIR=~/tmp/e2e-test AVALANCHEGO_BUILD_PATH=~/tmp/e2e-test/avalanchego ./scripts/install_avalanchego_release.sh
-./scripts/build.sh ~/tmp/e2e-test/avalanchego/plugins/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
-```
-
-Then, in the root of the `awm-relayer` project, run:
+E2E tests are ran as part of CI, but can also be ran locally with the `--local` flag. To run the E2E tests locally, you'll need to install Gingko following the intructions [here](https://onsi.github.io/ginkgo/#installing-ginkgo)
```

Comment:
can you remove the extra `---`'s above

Reply:
I'm not sure what you're referring to. Can you propose a diff in the PR?

Reply:
it's outside teh PR changes so can't propose diff, I meant here https://github.com/ava-labs/awm-relayer/blob/main/README.md?plain=1#L52